composer.json

```diff
@@ -63,10 +64,11 @@
       "phpcbf -p"
     ],
     "makepot-audit": [
-      "wp i18n make-pot . --exclude=\".github,.wordpress-org,bin,sample-data,node_modules,tests\" --slug=woocommerce"
+      "wp i18n make-pot . --include=\"packages\/woocommerce-blocks,packages\/woocommerce-rest-api,packages\/action-scheduler\" --exclude=\".github,.wordpress-org,bin,sample-data,node_modules,tests,packages\" --slug=woocommerce"
```

Comment:
If running only `makepot-audit` will not coombine files from WooCommerce Admin.
Also probably easy to just remove WooCommerce Admin instead of including all packages.

Reply:
@claudiosanches The reason I did it that way was it significantly reduced the memory required to run the command. If the command includes the `packages` folder with or without the WC Admin exclusion I get an OOM error. @peterfabian tested and found the memory required ranged around 360M. This implementation of the command requires less than 128M.